syftlib/src/main/java/org/openmined/kotlinsyft/WebRTC.kt

```diff
@@ -0,0 +1,304 @@
+package org.openmined.kotlinsyft
+
+import android.util.Log
+import org.openmined.kotlinsyft.Constants.WEBRTC_JOIN_ROOM
+import org.webrtc.*
+import org.webrtc.PeerConnection.*
+import java.nio.ByteBuffer
+
+
+typealias SDP_type = SessionDescription.Type
+
+private const val TAG = "WebRTCClient"
+
+open class WebRTCClient(
```

Comment:
Move this class/file to its own package `webrtc`

Reply:
It's a single file. Do we need a new package in that case? Unless we Separate Observers

Reply:
We already know there are other components that will be in the library. Better to have it in its own package.

Reply:
if we separate Observers then this will be needed

Reply:
There is an intention in naming a package too.